lib/argument.js

```diff
@@ -47,6 +50,24 @@ var argumentValidation = {
     if (!callback || !_.isFunction(callback)) {
       throw new Error('acquireToken requires a function callback parameter.');
     }
+  }, 
+
+  validateUserCodeInfo : function(userCodeInfo) {
+     if (!userCodeInfo){
+        throw new Error('The userCodeInfo parameter is required');
+     }
+
+     if (!userCodeInfo.hasOwnProperty(userCodeResponseFields.DEVICE_CODE)){
```

Comment:
Ah, I made a typo in my comment. I meant to say that you could get an error `Object userCodeInfo has no method 'hasOwnProperty'` because not everything will always have the method `hasOwnProperty`. But the chances of that happening are very very low, so you can probably ignore me here :)